Say Their Names/Source/Controller/Donations/DonationsViewCode.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  DonationsViewCode.swift
+//  Say Their Names
+//
+//  Created by Hakeem King on 6/3/20.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import UIKit
+
+/// The UI for Donations
+final class DonationsViewCode: UIView {
+    
+    private lazy var donationsLabel: UILabel = {
```

Comment:
tiny thing - this can be a let, it's anyway first accessed from `init` so not really getting any laziness 

```suggestion
    private let donationsLabel: UILabel = {
```

Reply:
I love these small comments. I am an Obj-c dev, so the small things in Swift I miss.